src/reactviews/pages/SchemaCompare/components/SchemaDifferences.tsx

```diff
@@ -41,6 +94,20 @@ const SchemaDifferences = ({ onDiffSelected }: Props) => {
         return nameParts.join(".");
     };
 
+    const handleIncludeExcludeNode = (diffEntry: DiffEntry, include: boolean) => {
+        if (diffEntry.position !== undefined) {
+            context.includeExcludeNode(diffEntry.position, diffEntry, include);
+        }
+    };
+
+    const handleIncludeExcludeAllNodes = () => {
+        if (allDiffsExcluded || someDiffsExcluded) {
+            context.includeExcludeAllNodes(true);
```

Comment:
can you add some inline comments here to spell out what the true/false refers to?
```
context.includeExcludeAllNodes(true /* include all */);
context.includeExcludeAllNodes(false /* exclude all */);
```